src/centos/scripts/start-marklogic.sh

```diff
@@ -13,6 +13,38 @@
 
 cd ~
 
+################################################################
+# Read in ENV values
+################################################################
+
+# If an ENV value exists in a list Append it the /etc/marklogic.conf file 
```

Comment:
...list, append it to the ....

Reply:
8850cb5f8497deddba22339da2962ad3a1184a69 updated